### ISSUE TITLE
Errors download Windows

### ISSUE THREAD

**Issue by ancucchi** [quickemu-project/quickemu#938]
When I try to "quickget" any of windows (8.1, 10 or 11) the system go in error and show the following:

`quickget windows 11`
```
Downloading Windows 11...
 - Product edition ID: 2935
 - SKU ID: 17442
 - Got latest ISO download link (valid for 24 hours): https://software.download.prss.microsoft.com/dbazure/Win11_23H2_English_x64v2.iso?t=5afc583f-4e2d-44e9-ba73-946c0645d1fe&P1=1708342105&P2=601&P3=2&P4=Ntu%2f6BHdZrFhHdi7J8DRzbS3csaY8EmMZ96gPQ1R9JejSllmjMwJUYVgWpS85hgIRHGTNIWRLwpWPc9JhLR%2fon4nU6%2bLQcZZ4VEfJDEgYTTtgXxCfJ1beKrmyv1gxHi2UPkQI8okhIUhZVYDU88lWPtcJ7oAu8gDvCgsZN08uLBuk5XlJ65U33J2W7BRoOPSJTx2iysWbj1KuzMQQCRk7l2yGoMtYK6X0YmJRJkcvrp9OYelZuSrE76G%2frlIK%2fPVc5FOxnyLQs%2f2wiSqNHYjR%2bA3HvOtDP50WBZ%2fPb%2fNfApedm8d3mOyQ5J1iIO39HS8clBm5ydqSYl6YHHnQpIshA%3
ERROR! Failed to download https://software.download.prss.microsoft.com/dbazure/Win11_23H2_English_x64v2.iso?t=5afc583f-4e2d-44e9-ba73-946c0645d1fe&P1=1708342105&P2=601&P3=2&P4=Ntu%2f6BHdZrFhHdi7J8DRzbS3csaY8EmMZ96gPQ1R9JejSllmjMwJUYVgWpS85hgIRHGTNIWRLwpWPc9JhLR%2fon4nU6%2bLQcZZ4VEfJDEgYTTtgXxCfJ1beKrmyv1gxHi2UPkQI8okhIUhZVYDU88lWPtcJ7oAu8gDvCgsZN08uLBuk5XlJ65U33J2W7BRoOPSJTx2iysWbj1KuzMQQCRk7l2yGoMtYK6X0YmJRJkcvrp9OYelZuSrE76G%2frlIK%2fPVc5FOxnyLQs%2f2wiSqNHYjR%2bA3HvOtDP50WBZ%2fPb%2fNfApedm8d3mOyQ5J1iIO39HS8clBm5ydqSYl6YHHnQpIshA%3 with wget. Try running 'quickget' again.
```

```
lsb_release --all
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.4 LTS
Release:	22.04
Codename:	jammy
```

```
uname -a
Linux UM-2021 5.15.0-94-generic #104-Ubuntu SMP Tue Jan 9 15:25:40 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
```
____________
**_Ancucchi_**

**Comment by ClaGre** [comment]
Yesterday I've the same error; when I've retry it works.
Then my suggestion is: retry

**Comment by gmantoha** [comment]
Same here. Retry doesn't help.
When trying the Link with wget directly it says:
`HTTP request sent, awaiting response... 403 Forbidden
2024-02-19 08:37:39 ERROR 403: Forbidden.`

**Comment by TuxVinyards** [comment]
I think we need an **_FAQ_** added to the quickemu readme for this.   If you search the closed issues as well as the open ones then the answers are there.

If you keep retrying, and from the same IP address, then Microsoft will block you ....

See recent resolved and closed issues here:

<https://github.com/TuxVinyards/qqX/issues/5>

<https://github.com/quickemu-project/quickemu/issues/898>

**Comment by flexiondotorg** [comment]
Closed via #986
